sherlock/resources/data.json

```diff
@@ -1530,6 +1530,14 @@
     "username_claimed": "blue",
     "username_unclaimed": "noonewouldeverusethis"
   },
+  "TETR.IO": {
+    "errorMsg": "No such user!",
+    "errorType": "message",
+    "url": "https://ch.tetr.io/api/users/{}",
```

Comment:
Since this is an API endpoint, use `urlProbe` and then for `url` put `https://ch.tetr.io/u/{}`

Reply:
Ohh, so that's how you do it. Pushing changes now.